app/views/planning_applications/_policy_consideration_list.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="govuk-accordion__section">
+  <div class="govuk-accordion__section-header">
+    <h3 class="govuk-accordion__section-heading">
+      <button type="button" id="accordion-default-heading-1" aria-controls="accordion-default-content-2" class="govuk-accordion__section-button" aria-expanded="false">
+        Proposal Details
+        <span class="govuk-accordion__icon" aria-hidden="true"></span></button>
+    </h3>
+  </div>
+  <div id="accordion-default-content-2" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
+    <ol class="govuk-body" style="line-height: 2;">
+      <% policy_considerations.order(created_at: :asc).each do |policy_consideration| %>
+        <%= render "question_answer_pair", policy_consideration: policy_consideration %>
```

Comment:
This may be an issue with my sample data, but I am currently seeing no proposal details displaying. I will investigate further and will delete this comment if it's just a local issue

Reply:
As an update to the point above, this is not an issue related to this PR  - it is a pre-existing issue with the Swagger data, so I'm going to resolve.

Reply:
Thanks, I would like to fix the swagger data though! Could you create a ticket?